client/src/components/LevelLayout.tsx

```diff
@@ -37,11 +52,14 @@ const LevelLayout: React.FC<Props> = ({
           Previous Step
         </Button>
         {game.stepIndex === game.steps.length - 1 && game.level < 5 - 1 ? (
+            <Flex>
```

Comment:
nit: From my understanding, the `Confetti` component doesn't need any layout mechanisms (`Flex` or `Grid`). You can replace this with a fragment (`<React.Fragment />` or `<> </>`).

Reply:
I switched it to empty tags